data/net/minecraft/server/commands/data/DataCommands.mapping

```diff
@@ -33,6 +33,101 @@ CLASS net/minecraft/server/commands/data/DataCommands
 	METHOD getSingletonSource (Lcom/mojang/brigadier/context/CommandContext;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;)Ljava/util/List;
 		ARG 0 context
 		ARG 1 dataProvider
+	METHOD lambda$decorateModification$22 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 3 context
+	METHOD lambda$decorateModification$23 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 3 context
+	METHOD lambda$decorateModification$24 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/builder/ArgumentBuilder;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 3 subArgBuilder
+	METHOD lambda$decorateModification$25 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 2 dataManipulator
+	METHOD lambda$decorateModification$26 (Ljava/lang/String;)Ljava/lang/String;
+		ARG 0 str
```

Comment:
I don't like `str` as a arbitrary parameter name, it tells very little (other than the type, badly), and is a abbreviation. For all the similarly named parameters in this class.